README.md

```diff
@@ -121,7 +126,7 @@ By adding data a block of code will be added to the notebook. The code will auto
 
 ![Generated code to handle Pandas DataFrame](doc/source/images/wml-5-generated-code-dataframe.png)
 
-> **IMPORTANT**: Don't forget to update the next cell `df = df1` (or whatever your dataframe number is) with the variable from the generated code, in the case of `Remote` Virtualized data. If you are using the `Local` data, use `df = df_data_1` (or whatever your dataframe number is).
+> **IMPORTANT**: Don't forget to update the next cell to assign the `df` variable. In the case of Virtualized data, it will look like `df = data_df_1`, `data_df_2, etc. For a locally uploaded file, it will look like `df = df_data_1` or `df_data_X` depending on your number `X` for the variable from the generated code.
```

Comment:
missing backtick after data_df_2.  The formatting is messed up.

Reply:
done